src/desktop/version.cpp

```diff
@@ -151,6 +151,22 @@ std::string os_version()
 		if(!ver.empty()) {
 			return ver;
 		}
+	} else {
```

Comment:
It's better to not use `else` for this, in case `/etc/lsb_release` is empty (see the check right above).

Reply:
I see, that's true